pkg/cluster/context.go

```diff
@@ -188,10 +189,34 @@ func (c *Context) Create(cfg *config.Config, options ...CreateOption) error {
 		return err
 	}
 
-	fmt.Printf(
-		"Cluster creation complete. You can now use the cluster with:\n\nexport KUBECONFIG=\"$(kind get kubeconfig-path --name=%q)\"\nkubectl cluster-info\n",
-		cc.Name(),
-	)
+	// TODO: consider shell detection.
```

Comment:
although, probably best to just to print these 3.

Reply:
that SGTM, 100% agree. shell detection could be nice if feasible, but isn't really necessary and this should be pretty helpful